HW1/P2/P2b.py

```diff
@@ -1,3 +1,32 @@
 from P2 import *
+import itertools
+import numpy as np
+import matplotlib.pyplot as plt
+from random import shuffle
 
-# Your code here
+import findspark
+findspark.init()
+from pyspark import SparkContext, SparkConf
+
+conf = SparkConf().setAppName('P2').setMaster('local')
+sc = SparkContext(conf=conf)
+
+# Calculate a mandelbrot set in parallel manner (with load balancing)
+
+data = list(itertools.permutations(range(1, 2001), 2))
```

Comment:
nice work, the histogram below threw me off at first but I realized the axis was different than mine (I plotted load against each partition). Our partitioning results are quite similar.